### PR TITLE
Fix: Show user popover for user avatars in activity table

### LINKED ISSUE #3702: [Activity - QA] Avatar click does not open relevant members popover in Activity feed
## Steps to reproduce

1. Navigate to the Activity Feed.
2. Locate an activity item and click on the avatar of the action creator.
3. Observe whether the members popover opens.

## Expected behaviour

* Clicking on the avatar should open a popover displaying relevant information about the action creator.

## Actual behaviour

* The popover does not open when clicking on the avatar, preventing users from accessing the member details.

image

image

### PR DESCRIPTION
**Pull request by rumzledz**
## Description

activity-popover

## Testing

1. Connect Leela's wallet
2. Make a Simple Payment
3. Visit the Activity page
4. Click on Leela's avatar
5. Verify that the user popover comes up

Resolves #3702

### CODE REVIEW

File: src/components/common/ColonyActionsTable/partials/ActionDescription/ActionDescription.tsx [hunk 1]
```diff
@@ -82,15 +83,23 @@ const ActionDescription: FC<ActionDescriptionProps> = ({
...
-              size={26}
-              userAddress={walletAddress}
-              userName={user?.profile?.displayName ?? undefined}
-              userAvatarSrc={
-                user?.profile?.thumbnail ?? user?.profile?.avatar ?? undefined
-              }
-              className="flex-shrink-0 flex-grow-0"
-            />
+            <UserInfoPopover
+              walletAddress={walletAddress}
```
Comment (mmioana) [inline src/components/common/ColonyActionsTable/partials/ActionDescription/ActionDescription.tsx:87]: Maybe here we want to set also `withVerifiedBadge={false}`

**Review by mmioana — CHANGES_REQUESTED** [review]
Nice work on this issue @rumzledz 🌟 

Now clicking a user avatar in the activity table shows up the user popover 💯 

https://github.com/user-attachments/assets/03561cc7-214e-4c74-b45c-269957b1016a


https://github.com/user-attachments/assets/94263719-db31-4bf9-b1d7-7cc8f8cd5116

The only reason I'm rejecting this is due to showing the `Verified` badge which shouldn't be displayed according to `Figma`
Screenshot 2024-11-28 at 09 21 21

File: src/components/common/ColonyActionsTable/partials/ActionDescription/ActionDescription.tsx (hunk 1 above)
Comment (rumzledz) [reply src/components/common/ColonyActionsTable/partials/ActionDescription/ActionDescription.tsx:87]: Ah good one! 👌

**Review by mmioana — APPROVED** [review]
Supaaa-fast with the fix once again @rumzledz  🐎 

All good from my side 🥇 

https://github.com/user-attachments/assets/378dec27-5232-465e-8dc4-60dd3ec88dc2

**Review by Nortsova — APPROVED** [review]
Nice one 😎 
User popover is on its place 👏 
image

**Pull request merged**